.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=--init-script /home/jw/.config/Code/User/globalStorage/redhat.java/1.10.0/config_linux/org.eclipse.osgi/51/0/.cp/gradle/init/init.gradle --init-script /home/jw/.config/Code/User/globalStorage/redhat.java/1.10.0/config_linux/org.eclipse.osgi/51/0/.cp/gradle/protobuf/init.gradle
```

Comment:
Please remove this file and filter `.settings` in `.gitignore`.

Reply:
Also I would advise to give IntelliJ IDEA a shot, you may like it more than Eclipse.